### DISCUSSION TITLE
Quiz - Tidak bisa restart Quiz

### DISCUSSION THREAD

**Discussion by moogli90** [teamalgoritma/community#211] [Q&A] [answered]
Mohon dibantu, ketika selesai quiz P4DS & PS, sy tidak bisa klik restart quiz dan lgsg diarahkan ke halaman cerificate..sy liat di classromm ada jg peserta lain yg bertanya hal yg sama, namun blm ada feedback, mohon solusinya

**Comment by kevbow** [comment, marked as answer]
Dear @moogli90,

Apabila Anda sudah mencapai/melebihi batas yang ditentukan dan dinyatakan lulus maka Anda tidak dapat mengulang quiz walaupun kesempatan kedua belum dipakai. Juga perlu kami informasikan kembali bahwa penilaian quiz bersifat biner, artinya jika Anda dinyatakan lulus maka Anda mendapatkan nilai sempurna (pada kasus ini berarti 4 poin) sementara jika Anda tidak lulus maka poin Anda 0.
